Examples/UIExplorer/js/NativeAnimationsExample.js

```diff
@@ -154,19 +201,27 @@ class InternalSettings extends React.Component {
             require('JSEventLoopWatchdog').install({thresholdMS: 25});
             this.setState({busyTime: '<none>'});
             require('JSEventLoopWatchdog').addHandler({
-              onStall: ({busyTime}) => this.setState((state) => ({
-                busyTime,
-                filteredStall: (state.filteredStall || 0) * 0.97 + busyTime * 0.03,
-              })),
+              onStall: ({busyTime}) =>
+                this.setState(state => ({
+                  busyTime,
+                  filteredStall: (state.filteredStall || 0) * 0.97 +
+                    busyTime * 0.03,
+                })),
             });
           }}
           onDisable={() => {
             console.warn('Cannot disable yet....');
           }}
         />
-        {this.state && <Text>
-          JS Stall filtered: {Math.round(this.state.filteredStall)}, last: {this.state.busyTime}
-        </Text>}
+        {this.state &&
+          <Text>
+            JS Stall filtered:
+            {' '}
+            {Math.round(this.state.filteredStall)}
+            , last:
```

Comment:
Maybe like this?

    <Text>
      {`JS Stall filtered: ${Math.round(this.state.filteredStall)}, `}
      {`last: ${this.state.busyTime}`}
    </Text>